admin/admin-classic-editor.php

```diff
@@ -190,6 +192,8 @@ public function post_lang_choice() {
 			wp_die( 'You are not allowed to edit this post.' );
 		}
 
+		( new User() )->can_translate_or_die( $lang );
+
```

Comment:
I suggest to move the code higher just after we check if `$lang` is empty.

Reply:
Sure, done in d0572fd5.